packages/quantic/force-app/solutionExamples/main/lwc/exampleInsightPanel/resultTemplates/defaultResultTemplate.html

```diff
@@ -1,38 +1,80 @@
 <template>
-  <c-quantic-result-template
-    is-any-preview-open={isAnyPreviewOpen}
-    result-preview-should-not-be-accessible={resultPreviewShouldNotBeAccessible}
+  <div
+    class="result-action_container slds-is-relative slds-var-p-horizontal_medium"
   >
-    <c-quantic-colored-result-badge
-      slot="label"
-      label={result.raw.filetype}
-    ></c-quantic-colored-result-badge>
-    <div
-      slot="badges"
-      class="slds-grid slds-grid_vertical-align-center slds-m-vertical_xx-small slds-m-right_small"
+    <c-quantic-result-template
+      hide-border
```

Comment:
We don't have that in our template, why the `hide-border` ?

Reply:
you are right that's not needed I will remove,
The quantic result template component automatically avoid displaying the border when the slot children is defined in the template indicating the presence of the folding feature so no need set this property to true, I will delete it.

Reply:
Not sure I understand the difference, do you need it or not need it? :D